plugins/polio/models/chronogram.py

```diff
@@ -78,6 +104,29 @@ def valid(self):
         return self.filter(deleted_at__isnull=True)  # Hide soft deleted items.
 
 
+class ChronogramTaskManager(models.Manager):
+    def get_queryset(self):
+        return (
+            super()
+            .get_queryset()
+            .select_related("chronogram__round")
+            .annotate(
+                annotated_deadline_date=ExpressionWrapper(
+                    F("chronogram__round__started_at") + F("start_offset_in_days"), output_field=DateField()
+                ),
+                # A negative delay in days means that the task is delayed.
+                annotated_delay_in_days=Case(
+                    When(status=self.model.Status.DONE, then=Value(0)),
+                    default=ExpressionWrapper(
```

Comment:
I'll keep a bookmark on that one for future reference. I've never implemented a sort on a calculated column in django.